### ISSUE TITLE
Add support to edit "Keywords" & "Subject" fields in document info

### ISSUE THREAD

**Issue by Nokia808** [jeromerobert/pdfarranger#237]
Hi.

Currently, it is possible to edit "Title", "Author", .... by pdfarranger. 

But there is other field called "Keywords" which is currently neither editable nor even visible by pdfarranger! Can you, kindly, support viewing & editing this field ?

**Comment by jeromerobert** [comment]
Could you (or someone) attach a small PDF file with some keywords ?

**Comment by Nokia808** [comment]
@jeromerobert 
Hi. I changed title because there is other field not included also ! It is the field of "Subject" !

Regarding a PDF example, see this:
[meta-data-example.pdf](https://github.com/jeromerobert/pdfarranger/files/4974978/meta-data-example.pdf)
It is only one page that I extract it from original PDF that I can not upload it here due to it's size. It contain the following fields:
- title,
- author (creator),
- subject,
- Keywords,

**Comment by Nokia808** [comment]
@jeromerobert 
Hi. I prepared other PDF example, but with meta data written in Latin characters (English) - see it bellow:
[other-example-en-meta-data.pdf](https://github.com/jeromerobert/pdfarranger/files/4986577/other-example-en-meta-data.pdf)

& bellow screenshot from a Windows program showing how meta data appearing & I added red arrows pointing for "subject" & "keywords" fields:
![Capture](https://user-images.githubusercontent.com/25375330/88632011-93224f00-d0a2-11ea-85cd-2c19676d6c8f.PNG)

**Comment by Nokia808** [comment]
@jeromerobert 
Hi dear. Just few minutes ago, I got version 1.6.2 flatpak from FlatHub.
I tested this issue with new version & it does not work currently: no support "Keywords" nor "Subject" fields - see bellow screenshoot:

![not-working](https://user-images.githubusercontent.com/25375330/89110801-f06d2600-d43d-11ea-9395-fef15358d2d0.png)

I think there is some confusion had been happened: you closed this issue for a commit related to issue for version 1.6.1 while this issue is not fixed by that commit because it is labelled to version 1.7 !

Kindly, review the state of this issue. Is it really fixed in master but not released in 1.6.1 & 1.6.2 or it is not fixed at all ! How a commit fb5dbb2 fix issue in version 1.6.2 & is also referred to fix 2nd issue #237 & this 2nd issue (issue #237) not fixed in version 1.6.2 ? Forgive me I'm not technician ......
